src/App.js

```diff
@@ -1,25 +1,28 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+// import pages
+import Home from './pages/Home';
+import About from './pages/About';
+import SingleCocktail from './pages/SingleCocktail';
+import Error from './pages/Error';
+
+// import components
+import Navbar from './components/Navbar';
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+    return (
+        <div>
+            <Router>
+                <Navbar />
+                <Routes>
+                    <Route path="/" element={<Home />} />
+                    <Route path="about" element={<About />} />
+                    <Route path="cocktail/:id" element={<SingleCocktail />} />
+                    <Route path="*" element={<Error />} />
+                </Routes>
+            </Router>
+        </div>
+    );
```

Comment:
maybe adding a footer here?

Reply:
Thanks for pointing this out, its implemented now. Thanks